[#12] profile별 application.yml 구분

mahmunsen opened the pull request:
###  profile별 application.yml 구분

- profile은 local, prod 두 개로 설정했습니다.

mahmunsen committed:
#5 [chore] jpa 사용 위한 설정

mahmunsen committed:
#5 [chore] jpa 사용 위한 설정

mahmunsen committed:
#5 [chore] jpa 쓰기 위한 dependency 추가 및 jasypt 추가

mahmunsen committed:
#5 [chore] jpa 쓰기 위한 dependency 추가 및 jasypt 추가

mahmunsen committed:
#5 [docs] JasyptConfig 작성
- secret_key_source 암호화 위한 configuration 생성

mahmunsen committed:
#5 [docs] JasyptConfig 작성
- secret_key_source 암호화 위한 configuration 생성

mahmunsen committed:
#5 [feat] product, category 엔티티 생성

mahmunsen committed:
#5 [feat] product, category 엔티티 생성

mahmunsen committed:
#5 [feat] PageResponse 생성
- 페이징 응답 위한 클래스

mahmunsen committed:
#5 [feat] PageResponse 생성
- 페이징 응답 위한 클래스

mahmunsen committed:
#5 [feat] 커서 페이지네이션(jpa없이) 간략 구현

mahmunsen committed:
#5 [feat] 커서 페이지네이션(jpa없이) 간략 구현

mahmunsen committed:
#5 [feat] jpa 사용 위한 리포지토리 생성

mahmunsen committed:
#5 [feat] jpa 사용 위한 리포지토리 생성

mahmunsen committed:
Merge remote-tracking branch 'origin/feature/4_cursorPaging' into feature/4_cursorPaging

mahmunsen committed:
#5 [refactor] jpa 없는 커서 페이지네이션 간략 구현 수정
- 추후 jpa 이용한 커서 페이지네이션 부분과 구분하기 위해 아래 클래스 이름 변경
- 기존 엔티티 이름 Product -> ProductV1
- 기존 서비스 이름 ProductService -> ProductV1Service 로 변경 등

mahmunsen committed:
#3 [refactor] 기능별 폴더 안 도메인별 package 제거

mahmunsen committed:
#6 [feat] profile별 application.yml 설정

f-lab-lyan commented on god-care-api/src/main/resources/mysql.yml:
application-mysql-prod.yml

mahmunsen committed:
#6 [refactor] profile별 yml 설정 수정
- deprecated된 include 대신 group 사용
- mysql.yml -> application-mysql-local.yml, application-mysql-prod.yml로 변경

f-lab-lyan commented on god-care-api/src/main/java/com/godcare/api/config/JasyptConfig.java:
```diff
@@ -0,0 +1,31 @@
+@Configuration
+public class JasyptConfig {
+
+    @Value("${secret-key-source}")
```
@Value 말고 @ConfigurationProperties를 적용해 보시면 더 좋을 것 같습니다.

mahmunsen replied:
넵, 어노테이션에 관해 찾아보고 적용하겠습니다.

f-lab-lyan commented on god-care-api/src/main/resources/application.yml:
```diff
@@ -1,7 +1,13 @@
-
-
+  profiles:
+    default: local
```
👍

f-lab-lyan reviewed (approved):
전반적으로 괜찮은 것 같습니다. 코멘트는 다른 PR에서 진행해보죠. :-)
